### DISCUSSION TITLE
请问如何将Alist在windows10系统中卸载重新安装？How do I uninstall and reinstall Alist in windows10?

### DISCUSSION THREAD

**Discussion by Hrzqzr** [AlistGo/alist#2470] [Q&A]
因为我的整个配置都乱套了，打算重新安装试试。
Because my whole configuration is messed up, I'm going to try to reinstall it.

**Comment by ghost** [comment]
不知道

**Comment by fiercec** [comment]
这也太奇怪了，怎么卸载是个问题

**Comment by Cheney1941** [comment]
直接删除alist.exe 所在的文件夹即可。然后你重新下载安装就行了。超级简单
